src/mlpack/methods/reinforcement_learning/noise/gaussian.hpp

```diff
@@ -45,7 +45,7 @@ class GaussianNoise
    */
   arma::colvec sample()
   {
-    return randi<arma::colvec>(size, arma::distr_param(mu, sigma));
+    return randn<arma::colvec>(size, arma::distr_param(mu, sigma));
```

Comment:
I guess this actually has to be

```suggestion
    return sigma * randn<arma::colvec>(size) + mu;
```

to account for older Armadillo versions. :+1:

Reply:
what bothers me is that I saw that I needed to do what you proposed when I saw the code, and then got distracted and forgot :smile: and ended up changing the letter only :smiling_face_with_tear: